EDU-18598 - Feedback #5357 - Content improvement (#2792)
vtexdocs/dev-portal-content: docs/release-notes/2026-06-25-faststore-webops-self-service-git-provider-management.md

---
title: "FastStore WebOps: Self-service Git provider management"
slug: "2026-06-25-faststore-webops-self-service-git-provider-management"
type: added
excerpt: "FastStore WebOps now allows you to change the GitHub organization and repository connected to your project."
createdAt: "2026-06-25T00:00:00.000Z"
tags:
  - FastStore
  - WebOps
---

> FastStore WebOps now provides a self-service interface for changing the GitHub organization and repository connected to your project. This feature streamlines the process of moving FastStore projects between organizations or repositories, eliminating the need to open support tickets for repository migrations.

Reviewer: ```suggestion
FastStore WebOps now provides a self-service interface for changing the GitHub organization and repository connected to your project. This feature streamlines moving FastStore projects between organizations or repositories, eliminating the need to open support tickets for repository migrations.
```

## What has changed?

The **Integrations** tab in the FastStore WebOps dashboard now includes a **Git Provider** section where you can configure the GitHub organization and repository associated with your FastStore project.

> Before, moving a FastStore project to a different GitHub organization or repository required opening a ticket with VTEX Support and providing detailed information including repository IDs, installation IDs, and account details. The process involved multiple manual steps and coordination with the Support team.

Reviewer: ```suggestion
Before, moving a FastStore project to a different GitHub organization or repository required opening a ticket with VTEX Support and providing detailed information, including repository IDs, installation IDs, and account details. The process involved multiple manual steps and coordination with the Support team.
```

Now, you can change your project's connected repository directly from the WebOps dashboard. The **Git Provider** card allows you to:

- **Change the GitHub organization:** Switch your project to a different GitHub organization by clicking `Change account` and selecting the target organization.
- **Change the repository:** Move your project to a different repository within the same organization by selecting the target repository from the dropdown.

## Why did we make this change?

> This change was made to simplify the repository migration process and give developers more control over their FastStore project configuration.

Reviewer: ```suggestion
This change simplifies the repository migration process and gives developers more control over their FastStore project configuration.
```

The feature is available to all FastStore projects using WebOps that meet the required permissions (`Owner` role in VTEX Admin and `Admin` role in GitHub repositories).

> >⚠️ The repository change feature isn't available for private GitHub instances. If your project is connected to a private GitHub instance, the option will not be displayed.

Reviewer: ```suggestion
>⚠️ The repository change feature isn't available for private GitHub instances. The WebOps dashboard won't display the option if your project is connected to a private GitHub instance.
```
